packages/kbn-types/lib.js

```diff
@@ -0,0 +1,10 @@
+// TODO Strictly speaking it would be great to not include this function in
```

Comment:
😢

Reply:
Yeah, I know :/ I still think we can get away from it, but don't have any clear ideas right now except waiting for a built-in solution in TS to solve this at the type level.